coffee_cmd/src/main.rs

```diff
@@ -78,6 +78,19 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Search { plugin } => match coffee.search(&plugin).await {
+            Ok(val) => {
+                let val = val["search"].as_str().unwrap();
+                if val == "notfound" {
+                    term::markdown(&("unable to locate plugin ".to_owned() + &plugin));
```

Comment:
`term::markdown` is wrong!

There are `term::success` and other calls, please look at the code

Reply:
Yes. I modified the code to use these calls

Reply:
yes